selectrum.el

```diff
@@ -973,18 +973,25 @@ ARG has same meaning as in `previous-history-element'."
     (goto-char (minibuffer-prompt-end))))
 
 (defun selectrum-select-from-history ()
-  "Select a candidate from the minibuffer history."
+  "Select a candidate from the minibuffer history.
+
+If selectrum isn't active insert selected candidate."
   (interactive)
   (let ((selectrum-should-sort-p nil)
         (enable-recursive-minibuffers t)
         (history (symbol-value minibuffer-history-variable)))
     (when (eq history t)
       (user-error "No history is recorded for this command"))
-    (let ((result (selectrum-read "History: " history)))
+    (let ((result
+           (let ((selectrum-candidate-inserted-hook nil)
+                 (selectrum-candidate-selected-hook nil))
+             (selectrum-read "History: " history :history t))))
```

Comment:
Why not just `let*` all three of these variables at the same time?

Reply:
Because the hooks would also be bound during the `selectrum--exit-with` down below where they should run like normal for the selectrum command which started the history command

Reply:
Oh, I forgot that we run the hooks from there now. Thanks, seems like you've done the right thing.